ci/jenkins/test.sh

```diff
@@ -345,12 +345,16 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    if [[ $WIN_NAME == *"jumper"* ]]; then
-        govc snapshot.revert -vm ${WIN_NAME} win-initial
+    if [[ ${TESTCASE} =~ "containerd" ]]; then
+        if [[ $WIN_NAME == *"jumper"* || $WIN_NAME == *"win-image"* || ${TESTCASE} == *"userspace"* ]]; then
+            govc snapshot.revert -vm ${WIN_NAME} win-initial
+        else
+            govc snapshot.revert -vm ${WIN_NAME} pristine-win-initial
+        fi
     else
-        govc snapshot.revert -vm ${WIN_NAME} pristine-win-initial
+        govc snapshot.revert -vm ${WIN_NAME} win-initial
```

Comment:
Could you specify the scenario you intend to address in the else branch? Now all windows test should run with containerd.